dmutils/forms/dm_fields.py

```diff
@@ -0,0 +1,202 @@
+'''
+This module includes classes that should be used with WTForms within Digital Marketplace apps.
+
+They extend the classes found in `wtforms.fields`, and should be used instead of those.
+The advantage they provide is that they have been deliberately designed to be used with
+the Digital Marketplace frontend toolkit macros. If a particular field is missing, it
+should be added to this file.
+
+The main functionality is provided by the mixin class, `DMFieldMixin`. When a derived class
+includes `DMFieldMixin` in its base classes then the following extra features are provided:
+
+        - a `hint` property that can be set in the initialiser
+        - a `question` property that contains the label text
+        - a `value` property for the data that is displayed
+        - an `error` property for the field validation error
+
+For more details on how `DMFieldMixin`, see the documentation in `dmutils/forms/mixins.py`.
+'''
+
+import datetime
+from itertools import chain
+
+import wtforms
+import wtforms.fields
+from wtforms.utils import unset_value
+from wtforms.validators import Length
+
+from .mixins import DMFieldMixin
+
+from .filters import strip_whitespace
+from .validators import EmailValidator
+
+
+__all__ = ['DMBooleanField', 'DMDecimalField', 'DMHiddenField', 'DMIntegerField',
+           'DMRadioField', 'DMStringField', 'DMEmailField', 'DMStripWhitespaceStringField',
+           'DMDateField']
+
+
+class DMBooleanField(DMFieldMixin, wtforms.fields.BooleanField):
+    pass
```

Comment:
From my experience in https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/798 we are missing some detail here required to make this work, specifically we need a `options ` I think.

Reply:
Got it, will add 👍